### ISSUE TITLE
Texture issue

### ISSUE THREAD

**Issue by Skabbety** [JarateKing/CleanTF2plus#13]
With Falt Textures enabled, I'm unable to see fencing. Included are two examples. Obviously I can't walk out of spawn on  the Blu team during the Pre-game timer is active on Payload maps, but it appears as if I can. The second screenshot is also an image where the wall is missing some fence aswell. I hope we can find a solution for this becuase this could be very annoying for CleanTF2+ users. And another reason I hope we can fix this is because, I do like the textures in the CleanTF2+ Mod. Thank You

![Evidence 1](https://user-images.githubusercontent.com/62624794/123184637-efbff280-d448-11eb-9c93-a73e8fbf9484.png)
![Evidence 2 ](https://user-images.githubusercontent.com/62624794/123184638-f0f11f80-d448-11eb-95cd-a2139346576e.png)

**Comment by micwoj92** [comment]
![image](https://user-images.githubusercontent.com/45581170/123211517-d90ab180-d4c3-11eb-9964-19bd05d07ef6.png)

**Comment by Skabbety** [comment]
Is this intended?

**Comment by Skabbety** [comment]
Is this intended?

**Comment by Skabbety** [comment]
Can I disable it manually?

**Comment by micwoj92** [comment]
> Is this intended?

Yes, because that's what the tool does.

> Can I disable it manually?

Yes, I think easiest way would be to just delete desired textures before packing into vpk.

**Comment by Skincanvas** [comment]
The missing texture in your example is metal/chicken_wire001.vtf. Unfortunately you will need to delete that texture from the flat textures vpk to get the intended fence feedback. Going to make a more comprehensive thread with a few problems I have found
